Google.PowerShell/Compute/GceFirewallCmdlets.cs

```diff
@@ -32,7 +40,7 @@ public class GetGceFirewallCmdlet : GceCmdlet
 
         /// <summary>
         /// <para type="description">
-        /// The name of the firewall rule to get.
+        /// The name of the firewall rule to get. -Name and -Firewall are aliases of this parameter.
```

Comment:
Why add the `-Alias` attribute? Shouldn't we prefer to have just one way to do things?